src/app/shared/links/links-list/pipes/link-type-icons.pipe.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Pipe, PipeTransform} from '@angular/core';
+import {LinkTypeModel} from '../../../../core/store/link-types/link-type.model';
+
+@Pipe({
+  name: 'linkTypeIcons'
+})
+export class LinkTypeIconsPipe implements PipeTransform {
+
+  public transform(linkType: LinkTypeModel): string[] {
+    return linkType.collections && linkType.collections.map(coll => coll.icon) || [];
```

Comment:
The same here. It does not make sense to create new custom pipes for a single use case if we can make use of the existing ones.